### ISSUE TITLE
Need cherry-pick #253 to release branch

### ISSUE THREAD

**Issue by shonge** [pingcap/tidb-lightning#257]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.
Use tidb-lightning and set tidb-password
```
kn t1 logs -f lighting-tidb-lightning-qjtfs
flag provided but not defined: -tidb-password
Usage:
  -L string
        log level: info, debug, warn, error, fatal (default "info")
  -V    print version of lightning
  -backend string
        delivery backend ("importer" or "mysql")
  -c string
        (deprecated alias of -config)
  -config string
        tidb-lightning configuration file
  -d string
        Directory of the dump to import
  -importer string
        address (host:port) to connect to tikv-importer
  -log-file string
        log file path
  -pd-urls string
        PD endpoint address
  -server-mode
        start Lightning in server mode, wait for multiple tasks instead of starting immediately
  -status-addr string
        the Lightning server address
  -tidb-host string
        TiDB server host
  -tidb-port int
        TiDB server port (default 4000)
  -tidb-status int
        TiDB server status port (default 10080)
  -tidb-user string
        TiDB user name to connect
Failed to parse command flags:  flag provided but not defined: -tidb-password
```
2. What did you expect to see?
Can set tidb-password

3. What did you see instead?
Failed to parse command flags:  flag provided but not defined: -tidb-password
4. Versions of the cluster

    - TiDB-Lightning version (run `tidb-lightning -V`):

        ```
       v3.0.8
        ```
        
    - TiKV-Importer version (run `tikv-importer -V`)

        ```
        (paste TiKV-Importer version here)
        ```
        
    - TiKV version (run `tikv-server -V`):

        ```
        (paste TiKV version here)
        ```

    - TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
        (paste TiDB cluster version here)
        ```

    - Other interesting information (system version, hardware config, etc):

        >
        >

5. Operation logs
   - Please upload `tidb-lightning.log` for TiDB-Lightning if possible
   - Please upload `tikv-importer.log` from TiKV-Importer if possible
   - Other interesting logs
   
6. Configuration of the cluster and the task
   - `tidb-lightning.toml` for TiDB-Lightning if possible
   - `tikv-importer.toml` for TiKV-Importer if possible
   - `inventory.ini` if deployed by Ansible

7. Screenshot/exported-PDF of Grafana dashboard or metrics' graph in Prometheus for TiDB-Lightning if possible

**Comment by kennytm** [comment]
The `release` branches have been synchronized and #253 will be included in v3.0.9.

**Comment by shonge** [comment]
Thank you.
